vector/src/main/java/im/vector/app/core/ui/views/PresenceStateImageView.kt

```diff
@@ -34,7 +35,7 @@ class PresenceStateImageView @JvmOverloads constructor(
 ) : AppCompatImageView(context, attrs, defStyleAttr) {
 
     fun render(showPresence: Boolean = true, userPresence: UserPresence?) {
```

Comment:
instead of the view internally knowing about the build config we can lift this up to https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt#L128 and make use of the existing `showPresence` 

the `RoomSummaryItemFactory` acts as an injection entry point so we could also inject the `MatrixConfiguration` into the factory and reuse the configuration